js/mage/adminhtml/input-counter.js

```diff
@@ -1,5 +1,5 @@
 // https://stackoverflow.com/a/44436408/5703627
-document.observe('dom:loaded', function() {
+window.addEventListener('load', function() {
```

Comment:
```suggestion
window.addEventListener('DOMContentLoaded', function() {
```
wouldn't DOMContentLoaded be more similar to the original dom:loaded?

Reply:
True.

From [docs](http://api.prototypejs.org/dom/document/observe/index.html#:~:text=DOMContentLoaded):
> On supporting browsers it fires on DOMContentLoaded and on unsupporting browsers it simulates it using smart workarounds.